test/integration/golden_response/get_triples/stat_var.json

```diff
@@ -464,7 +464,7 @@
       ],
       "predicate": "measurementDenominator",
       "objectId": "Count_Person_Female",
-      "objectName": "Count of Female",
+      "objectName": "Female",
```

Comment:
Female Population?

Reply:
I feel thats okay and consistent with other "Count of <ConstraintVal>", where only <ConstraintVal> shows up?  "Count of Person" alone is Population.